keras/random/seed_generator.py

```diff
@@ -98,6 +97,8 @@ def from_config(cls, config):
 
 
 def global_seed_generator():
+    from keras.utils import jax_utils
```

Comment:
This is problematic because it will be run regardless of backend and it will import JAX. We need to move `is_in_jax_tracing_scope` to a location that does not import JAX, or alternatively do an inline import of JAX for the JAX layer, or alternatively put the JAX layer in its own file (e.g. utils/jax_layer.py or something like that)

Reply:
Actually maybe moving the JAX layer to its own file is the better solution. We might need other JAX utils in the future, and one file = one component is a reasonable standard.

Reply:
Ok, will do. Still in the `keras/utils` folder though?

Reply:
Moved to `keras/utils/jax_layer.py`.